rpc.go

```diff
@@ -16,96 +16,22 @@ package libvirt
 
 import (
 	"bytes"
-	"encoding/binary"
 	"errors"
 	"fmt"
 	"io"
-	"net"
 	"reflect"
 	"strings"
 	"sync/atomic"
-	"unsafe"
 
 	"github.com/digitalocean/go-libvirt/internal/constants"
 	"github.com/digitalocean/go-libvirt/internal/event"
 	xdr "github.com/digitalocean/go-libvirt/internal/go-xdr/xdr2"
+	"github.com/digitalocean/go-libvirt/socket"
 )
 
 // ErrUnsupported is returned if a procedure is not supported by libvirt
 var ErrUnsupported = errors.New("unsupported procedure requested")
 
-// request and response types
-const (
```

Comment:
will moving these currently exported constants potentially impact any clients?  I would expect anyone, other than the code directly interacting with libvirt to need them, but they must be exported for some purpose.... possibly just for the unit tests?